audbackend/core/api.py

```diff
@@ -82,43 +157,27 @@ def create(
         backend object
 
     Raises:
-        BackendError: if an error is raised on the backend
+        BackendError: if an error is raised on the backend,
+            e.g. repository cannot be created
```

Comment:
If I saw it correctly inside the `Artifactory._create()` implementation it will also raise this error if the repo exists already, correct?
Then I think we should also mention this here explicitly:

```suggestion
            e.g. repository exists already
            or cannot be created
```